pipelines/HSC/DRP-RC2_subset.yaml

```diff
@@ -405,20 +181,6 @@ subsets:
       'tract' or 'patch' as part of the data ID expression if all
       reference catalogs or diffIm templates that cover these
       detector-level quanta are desired.
-  nightlyStep5:
```

Comment:
I think this change necessitates that the relevant lines in the `rc2_subset` repo are also removed:
https://github.com/lsst/rc2_subset/blob/1a133e3ce3b2ff950d60b48526d126463bd35fc6/bin/run_rc2_subset.sh#L119. If this isn't removed, then I'd expect any nightly test to fail when it gets to `nightlyStep5`.

Reply:
There's also mention of `nightlyStep5` inside the legacy `measureHscRC2Metrics.sh` script, which also calls `faro` at points. This script has been superceded by `run_rc2_subset.sh`, and I wouldn't object to it simply being removed at this point.

Reply:
Edit: please disregard the above. I clicked your PR links on the Jira ticket, but one them was duplicated and `rc2_subset` was missing, so I didn't realize there was a PR for `rc2_subset` as well!

I still wouldn't object if you'd rather just remove the entire `measureHscRC2Metrics.sh` script however, as it should probably not be used any longer.